sdk/documentintelligence/ai-document-intelligence-rest/src/index.ts

```diff
@@ -13,5 +13,6 @@ export * from "./outputModels.js";
 export * from "./paginateHelper.js";
 export * from "./pollingHelper.js";
 export * from "./utils.js";
+export { KnownDocumentIntelligenceAudience } from "./audience.js";
```

Comment:
why there is no api view update? i believe this enum should be newly added.

Reply:
added